src/go/rdctl/cmd/snapshotCreate.go

```diff
@@ -14,16 +12,14 @@ var snapshotCreateCmd = &cobra.Command{
 	Use:   "create <name>",
 	Short: "Create a snapshot",
 	Args:  cobra.ExactArgs(1),
-	RunE: func(cmd *cobra.Command, args []string) error {
-		return snapshotCreate(cmd, args)
-	},
+	RunE:  stopBackendCallFuncAndRestartBackend(createSnapshot),
 }
 
 func init() {
 	snapshotCmd.AddCommand(snapshotCreateCmd)
 }
 
-func snapshotCreate(cmd *cobra.Command, args []string) error {
+func createSnapshot(cmd *cobra.Command, args []string) error {
 	paths, err := paths.GetPaths()
```

Comment:
Again, please don't call this variable `paths`.

Reply:
Changed to importing `paths` package as `p`, as discussed.